packages/devextreme-themebuilder/tests/modules/logger.test.ts

```diff
@@ -6,7 +6,7 @@ describe('Logger tests', () => {
 
   beforeEach(() => {
     jest
-      .useFakeTimers('modern')
+      .useFakeTimers()
```

Comment:
BC in jest API - now `modern` timers are the default ones (and the timer type is specified differently)